frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/import-providers-tab/providers/vm-import-password.tsx

```diff
@@ -70,6 +72,17 @@ const stateToProps = (state, { wizardReduxID, provider }) => {
     OvirtProviderField.PASSWORD,
     VMWareProviderField.PASSWORD,
   );
+  const validationField = iGetProviderFieldAttribute(
+    state,
+    wizardReduxID,
+    provider,
+    'validation',
+    OvirtProviderField.API_URL,
+    VMWareProviderField.HOSTNAME,
+  );
+  const isValidationCritical =
+    iGet(validationField, 'type') === ValidationErrorType.Error ||
```

Comment:
let's use `[].includes` here for brevity

Reply:
fixed